### PR TITLE
contrib/gammastep: new package

### PR DESCRIPTION
**Pull request by jabuxas**

### CODE REVIEW

File: contrib/gammastep/template.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+pkgname = "gammastep"
+pkgver = "2.0.9"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = [
+    "autoconf",
```
Comment (nekopsykose) [inline contrib/gammastep/template.py:7]:
```suggestion
```

pulled by autoconf

also sort the *depends lists

File: contrib/gammastep/template.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+makedepends = [
+    "wayland-devel",
+    "libdrm-devel",
+    "libx11-devel",
+    "libxcb-devel",
+    "glib-devel",
+    "geoclue-devel",
+    "libxxf86vm-devel",
+]
+pkgdesc = "Redshift alternative with wayland support"
```
Comment (nekopsykose) [inline contrib/gammastep/template.py:25]:
```suggestion
pkgdesc = "Adjusts the color temperature of the screen"
```

File: contrib/gammastep/template.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+maintainer = "jabuxas <lucasbarbieri.c@gmail.com>"
+license = "GPL-3.0-or-later"
+url = "https://gitlab.com/chinstrap/gammastep"
+source = f"{url}/-/archive/v{pkgver}/gammastep-v{pkgver}.tar.bz2"
```
Comment (nekopsykose) [inline contrib/gammastep/template.py:29]:
```suggestion
source = f"{url}/-/archive/v{pkgver}.tar.gz"
```

shorter version of the same

also for github autotarballs .gz is preferred

Comment (jabuxas) [reply contrib/gammastep/template.py:29]: ok

File: contrib/gammastep/template.py (hunk 1 above)
Comment (jabuxas) [reply contrib/gammastep/template.py:7]: wdym pulled by autoconf?

Comment (nekopsykose) [reply contrib/gammastep/template.py]: i meant automake

**Pull request merged**